src/psyclone/transformations.py

```diff
@@ -79,12 +87,14 @@ class RegionTrans(Transformation):
     def _validate(self, node_list):
         '''Test if the nodes in node_list are in the original order.
 
-        :param list node_list: List of nodes.
-        :raises TransformationError: If the nodes in the list are not\
-                in the original order in which they are in the AST,\
+        :param list node_list: List of PSyIR nodes.
+        :raises TransformationError: If the nodes in the list are not \
+                in the original order in which they are in the AST, \
                 a node is duplicated or the nodes have different parents.
-        '''
+        :raises TransformationError: if any of the nodes to be enclosed in \
+                the region are of an un-supported type.
```

Comment:
s/un-supported/unsupported/g. I have seen "unsupported", not so sure about the hyphenated version. Or evene "... are not of a supported type"?

Reply:
Oops again - fixed.